### DISCUSSION TITLE
Table data not appearing

### DISCUSSION THREAD

**Discussion by MGW-SumathyV** [nocodb/nocodb#3730] [Q&A]
Hi,

I added the data in the table ,but it not showing up. It showing below error
![table data error](https://user-images.githubusercontent.com/70773267/191522052-21873f26-db00-473e-9862-7c9c79c65ca9.png)

Could you please let me know how to solve the issue

Thanks and Regards,
Sumathy Venugopal

**Comment by o1lab** [comment]
Please provide all details when raising the issue. Looks like this is an old version.

**Comment by MGW-SumathyV** [comment]
I found the issue there is a duplicate column in one of my tables.I removed that. Its working fine now.
Thank you for your quick response.
